doctr/__main__.py

```diff
@@ -60,9 +60,8 @@ def get_parser():
     deploy_parser.add_argument('--built-docs', default=None,
         help="""Location of the built html documentation to be deployed to
         gh-pages. If not specified, Doctr will try to automatically detect build location""")
-    deploy_parser.add_argument('--gh-pages-docs', default='docs',
```

Comment:
Does argparse let you set a default for positional arguments? You'd have to do the flag logic manually (use `parser.error` to print the error).